index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+ <head>
+   <title>The Gallery of Horns</title>
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ </head>
+ <body>
+   <header>
+        <h1>The Gallery of Horns</h1>
+        <select id="horny_animals">
+          <option value="default">Filter by Keyword</option>
+          <option value="narwhal">Narwhal</option>
+          <option value="rhino">Rhino</option>
+          <option value="unicorn">Unicorn</option>
+          <option value="unilego">Unilego</option>
+          <option value="triceratops">Triceratops</option>
+          <option value="markhor">Markhor</option>
+          <option value="mouflon">Mouflon</option>
+          <option value="addax">Addax</option>
+          <option value="chameleon">Chameleon</option>
+          <option value="lizard">Lizard</option>
+          <option value="dragon">Dragon</option>
+      </select>

```

Comment:
All of these options should be generated using the keyword property from the list of Animal objects, and not hard-coded into the index file. So once that feature is completed, the generated index page should look similar to this, but the data should not be hard-coded (just in case we have some different keywords for future data).